common/display/displayplanemanager.cpp

```diff
@@ -163,13 +157,7 @@ bool DisplayPlaneManager::ValidateLayers(
   if (layer_begin != layer_end) {
     auto overlay_end = overlay_planes_.end();
     if (cursor_plane_) {
-#ifdef DISABLE_CURSOR_PLANE
       overlay_end = overlay_planes_.end() - 1;
```

Comment:
If DISABLE_CURSOR_PLANE is not defined, seems this line should be ommited?

Reply:
Instead line 169 to 171 shoud remain

Reply:
165-167 is same with 169-171 since enable_last_plane_  has been removed

Reply:
Ok.